### PR TITLE
feat: tab mobile enhancement

### PR DESCRIPTION
**Pull request by ninariccimarie**

### CODE REVIEW

**Review by marthayaputra — APPROVED** [review]

File: src/@next/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -73,7 +90,13 @@ export const Tabs = React.forwardRef<HTMLDivElement, TabsProps>(function Tabs(
 
   return (
     <StyledTabsContainer ref={ref}>
-      <StyledTabHeaderContainer>
+      <StyledTabHeaderContainer
+        ref={tabsHeaderRef}
+        onScroll={handleScroll}
+        data-scroll-left={canScrollLeft}
+        data-scroll-right={canScrollRight}
+        data-scroll-both={canScrollLeft && canScrollRight}
```
Comment (marthayaputra) [inline src/@next/Tabs/Tabs.tsx]: Sorry missed this earlier, I think we can combine `data-scroll-left` and `data-scroll-right` : https://www.w3.org/TR/CSS2/selector.html#attribute-selectors

File: src/@next/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -39,9 +39,37 @@ export const Tabs = React.forwardRef<HTMLDivElement, TabsProps>(function Tabs(
...
   const tabLength = tabs.length;
 
   const [selectedTabIndex, setSelectedTabIndex] = useState(0);
+  const [canScrollLeft, setCanScrollLeft] = useState(false);
+  const [canScrollRight, setCanScrollRight] = useState(false);
+  const [isDragging, setIsDragging] = useState(false);
+  const [dragStartX, setDragStartX] = useState(0);
+  const [scrollLeft, setScrollLeft] = useState(0);
+
+  const handleMouseDown = (event: React.MouseEvent<HTMLDivElement>) => {
```
Comment (chiahou) [inline src/@next/Tabs/Tabs.tsx]: @ninariccimarie When I drag the tabs using the mouse, the behavior seems to be quite jerky. Does dragging prevent the click event from firing?

**Review by marthayaputra — APPROVED** [review]

File: src/@next/Tabs/TabStyle.ts [hunk 3]
```diff
@@ -63,7 +90,7 @@ export const StyledSpan = styled.span`
 
 export const StyledTabButton = styled.button`
```
Comment (chiahou) [inline src/@next/Tabs/TabStyle.ts:91]: @ninariccimarie The tab now has a blue-ish background color when clicked. Can we remove it?

**Review by chiahou — APPROVED** [review]
qc passed, thanks

**Pull request merged**